PostFreeSurfer/PostFreeSurferPipelineLongLauncher.sh

```diff
@@ -14,8 +14,9 @@ source "$HCPPIPEDIR/global/scripts/processingmodecheck.shlib" "$@" # Check proce
 opts_AddMandatory '--study-folder' 'StudyFolder' 'path' "folder containing all subjects" 
 opts_AddMandatory '--subject' 'Subject' 'subject ID' "subject label"
 opts_AddMandatory '--template' 'LongitudinalTemplate' 'template ID' "longitudinal template label (matching the one used in FreeSurferPipeline-long)"
-opts_AddMandatory '--timepoints' 'Timepoint_list' '@ separated list of timepoints (should match directory names)'
-opts_AddMandatory '--queuing-command' 'queuing_command' 'string' "queuing command"
+opts_AddMandatory '--timepoints' 'Timepoint_list' 'list' 'comma separated list of timepoints/sessions (should match directory names)'
+opts_AddMandatory '--parallel-mode' 'parallel_mode' 'string' "parallelization execution mode, one of FSLSUB, BUILTIN"
+opts_AddOptional '--parallel-mode-param' 'parallel_mode_param' 'custom' "FSLSUB: queue name [long.q]; BUILTIN: maximum number of threads [4]"
```

Comment:
The "type specifier" is just a string to display in the help info, try not to confuse the person reading it.
```suggestion
opts_AddOptional '--parallel-mode-param' 'parallel_mode_param' 'value' "FSLSUB: queue name [long.q]; BUILTIN: maximum number of threads [4]"
```

Reply:
This is obsolete, I think I fixed it but please check